src/components/VuetableRowHeader.vue

```diff
@@ -187,8 +190,29 @@ export default {
     },
 
     onColumnHeaderClicked (field, event) {
-      this.vuetable.orderBy(field, event)
+      if(this.vuetable.isDragged){
```

Comment:
Did it pass the lint test?

Reply:
There's no plugin installed for lint checks, so it never actually went through one.
Wrong indentation anywhere?